src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Brm-bm-bm! Machine started its work");
```

Comment:
better add the type of machine for all messages

Reply:
Ok, Thank you!